gui_qt/app.py

```
@@ -6560,51 +6740,96 @@ def _refresh_litellm_credential_status(self) -> None:
+                f"Provider 凭据 — {provider_display_label(provider)}"
+                if provider
+                else "Provider 凭据"
+            )
         saved_message = ""
-        if status.provider and status.provider != "ollama":
+        if provider and provider != "ollama":
             try:
                 saved_message = (
-                    "系统凭据管理器中已保存密钥。"
-                    if load_provider_api_key(status.provider)
+                    "系统凭据管理器中已保存密钥；如同时存在环境变量，请求优先使用此密钥。"
```

Ruff (0.15.21)

[warning] String contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF001)